tests/try_scraper/test_try_scraper.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+import unittest
+import urllib
+
+from mozdownload import TryScraper
+from mozdownload.utils import urljoin
+import mozhttpd_base_test as mhttpd
+
+firefox_tests = [
+    # -p mac64 --changeset=8fcac92cfcad
+    {'args': {'platform': 'mac64',
+              'changeset': '8fcac92cfcad'},
+     'target': '8fcac92cfcad-firefox-38.0a1.en-US.mac.dmg',
+     'target_url': 'firefox/try-builds/test-user@mozilla.com-8fcac92cfcad/'
+                   'try-macosx64/firefox-38.0a1.en-US.mac.dmg'},
+    # -p mac --changeset=8fcac92cfcad
+    {'args': {'platform': 'mac',
+              'changeset': '8fcac92cfcad'},
+     'target': '8fcac92cfcad-firefox-38.0a1.en-US.mac.dmg',
+     'target_url': 'firefox/try-builds/test-user@mozilla.com-8fcac92cfcad/'
+                   'try-macosx64/firefox-38.0a1.en-US.mac.dmg'},
+    # -a firefox -p linux64 --changeset=8fcac92cfcad
+    {'args': {'platform': 'linux64',
+              'changeset': '8fcac92cfcad'},
+     'target': '8fcac92cfcad-firefox-38.0a1.en-US.linux-x86_64.tar.bz2',
+     'target_url': 'firefox/try-builds/test-user@mozilla.com-8fcac92cfcad/'
+                   'try-linux64/firefox-38.0a1.en-US.linux-x86_64.tar.bz2'},
+    # -a firefox -p linux --debug-build
+    {'args': {'platform': 'linux',
+              'changeset': '8fcac92cfcad',
+              'debug_build': True},
+     'target': '8fcac92cfcad-debug-firefox-38.0a1.en-US.linux-i686.tar.bz2',
+     'target_url': 'firefox/try-builds/test-user@mozilla.com-8fcac92cfcad/'
+                   'try-linux-debug/firefox-38.0a1.en-US.linux-i686.tar.bz2'},
+    # -a firefox -p win32 --changeset=8fcac92cfcad
+    {'args': {'platform': 'win32',
+              'changeset': '8fcac92cfcad'},
+     'target': '8fcac92cfcad-firefox-38.0a1.en-US.win32.installer.exe',
+     'target_url': 'firefox/try-builds/test-user@mozilla.com-8fcac92cfcad/'
+                   'try-win32/firefox-38.0a1.en-US.win32.installer.exe'},
+    # -a firefox -p win64 --changeset=8fcac92cfcad
+    {'args': {'platform': 'win64',
+              'changeset': '8fcac92cfcad'},
+     'target': '8fcac92cfcad-firefox-38.0a1.en-US.win64-x86_64.installer.exe',
+     'target_url': 'firefox/try-builds/test-user@mozilla.com-8fcac92cfcad/'
+                   'try-win64/firefox-38.0a1.en-US.win64-x86_64.installer.exe'},
```

Comment:
This is using the old format of the filename. Given that try builds don't survive that long, change that to `win64.installer`. I don't see a reason to test for `x86_64`.